data/keystore/src/main/java/tech/pegasys/artemis/bls/keystore/model/Param.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.bls.keystore.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/** Placeholder for empty params */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Param {
```

Comment:
If this is just representing an empty param can just rename this to emptyParam to make it clearer

Reply:
done